queries.sql

```diff
@@ -14,4 +14,50 @@ SELECT * FROM animals WHERE neutered=true;
 
 SELECT * FROM animals WHERE NOT name='Gabumon';
 
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+SELECT * FROM animals;
+
+BEGIN;
+
+ALTER TABLE animals ADD species VARCHAR(50);
+
+UPDATE animals SET species='unspecified';
+
+ROLLBACK;
+
+UPDATE animals SET species='pokemon' WHERE species IS NULL;
```

Comment:
Please update the animals table by setting the species column to `digimon` for all animals that have a name ending in `mon`. Then commit the changes and kindly verify that change was made before and after `commit`.

Reply:
Thanks.Man

All the changes were made. Although I forgot to add them to the queries SQL file. 👍🏻